services/entity-server/src/views/data/data-view.js

```diff
@@ -22,7 +22,7 @@ export const DataView = () => {
   }
   return (
     <div className={style.dataViewWrapper}>
-      <PageTitle title={"Data"}/>
+      <PageTitle title={"Data"} />
```

Comment:
This seems to be another auto-format thing. I will adapt mine accordingly

Reply:
im using prettier without any mods
https://prettier.io

the idea is that you use the linter (in this case built into create react app) only for objective problems 

and then use an opinionated code formatter for everything that is subjective

Reply:
you can then integrate it into the file watcher in Webstorm, or whatever IDE you use

Reply:
nice, i will look into how to setup with webstorm